.eleventy.js

```diff
@@ -0,0 +1,19 @@
+const GIGSBOAT_INPUT_DIR = 'gigsboat-eleventy-starter'
```

Comment:
This is duplicated in [gigsboat.json](https://github.com/estherixz/gigs-eleventy-template/pull/1/files#diff-47d2124be86965a287de47884519918160f5f3f4d931cb24a19d84ba2d1f433eR35), I could either read from there or move the config to the cli.

The first option means we'll be doing an additional file read which I wasn't keen on and the second option means we'd take away the ability to configure `.eleventy.js`. Maybe there's a third option I haven't thought of.